packages/data-context/src/data/ProjectConfigManager.ts

```diff
@@ -135,6 +135,12 @@ export class ProjectConfigManager {
           ...loadConfigReply.requires,
           this.configFilePath,
         ])
+
+        // Only call "to{App,Launchpad}" once the config is done loading.
+        // Calling this in a "finally" would trigger this emission for every
+        // call to get the config (which we do a lot)
+        this.options.ctx.emitter.toLaunchpad()
```

Comment:
Nice, I noticed we have a lot of excess gql calls in general, hope we can find a good way to monitor/track these eventually... no idea how other apps handle this, or do they just have lots of requests 🤔 

Nice thing about REST is you are very deliberate about requests - just thinking out loud here, gql sure is nice but man it has some trade-offs (complexity, hard to really know what/when a call is triggered).

Reply:
I think it all boils down the the `toLaunchpad` calls. This causes every active query to retrigger. If properly designed, we could probably get away with removing this altogether. Or we could rely more heavily on subscriptions to update data.